### ISSUE TITLE
PostgreSQL - check constrains

### ISSUE THREAD

**Issue by Ostricker** [wise-coders/dbschema#5]
1. No errors.
2. PC - windows 10, DbSchema 8.2.7 build 200201. SERVER - Ubuntu 18, PostgreSQL 11.6 inside docker container. 
3. DbSchema does not recognize check constrains from Postgres.
a) I create check inside dbschema. It succesfully translates to database.
b) when i do "refresh local schema from database" - dbschema shows that the check is not present in database. Via database client (DataGrip) I can see that the check is there. 
c) when i create checks via script in the database - dbschema does not see any changes.

**Comment by wise-coders** [comment]
There is something we miss here. We use this query to reverse engineer constraints from postgres:

```
SELECT relname, conname, pg_get_constraintdef(conrelid)
FROM pg_constraint c 
JOIN pg_class cl ON cl.oid=conrelid 
JOIN pg_namespace nl ON nl.oid=relnamespace 
WHERE 
contype = 'c' AND nspname='${schema}'
```
The query is configurable in Project / Settings / Postgres Reverse engineer

If I test it, the constraint is listed, the definition ( pg_get_constraintdef(...) ) returns a null.
Maybe somebody from the community has an idea what is wrong in this query ?

To test I created this constraint:
ALTER TABLE "public".sample2 ADD CONSTRAINT ck1 CHECK ( age > 14 )

**Comment by Ostricker** [comment]
SELECT pg_get_constraintdef(co.OID)
FROM pg_namespace n
JOIN pg_class cl on (n.oid = cl.relnamespace)
JOIN pg_constraint co on (co.conrelid = cl.oid)
WHERE contype = 'c' and nspname='${schema}';

**Comment by wise-coders** [comment]
Thank you for the query. Now we implemented this in DbSchema, by tomorrow this change will be available under the version number 8.2.9, buildnumber 200330 or higher.
